core/integration/features/verified/0032-PRIM-price_monitoring_lognormal.feature

```diff
@@ -428,5 +428,5 @@ Feature: Price monitoring test using forward risk model (bounds for the valid pr
 
     And the market data for the market "ETH/DEC20" should be:
       | mark price | trading mode            | horizon | min bound | max bound | target stake | supplied stake | open interest |
-      | 1000000    | TRADING_MODE_CONTINUOUS | 60      | 994606    | 1005415   | 7434000      | 0              | 10            |
-      | 1000000    | TRADING_MODE_CONTINUOUS | 600     | 977751    | 1022678   | 7434000      | 0              | 10            |
+      | 1000000    | TRADING_MODE_CONTINUOUS | 60      | 994606    | 1005415   |   7434000    | 90000000       | 10            |
```

Comment:
What I mean is why we had 0 supplied stake before, but have something positive now.

Reply:
yes. before the probability was 0.0000038991101650020675 and now its 0.49811856653940937. because before it was being evaluated as being 100 tick sizes from the best and now it's being evaluated as being 100/100000 away from the best.